### DISCUSSION TITLE
Week 2: Data Collection and Scraping

### DISCUSSION THREAD

**Discussion by tayojabar** [AISaturdaysLagos/cohort_structure#6] [General]
Share questions and learnings from reading the material or perhaps what you pick from the session. 

Thanks.

**Comment by RiddyRichie** [comment]
Hi sir, what library will i have to install while using request as import because it showing an error that no module found on request.

**Reply by tejuafonja** [reply]
Hey Richie, 

Did you install the package yet? You need to do something like this `python -m pip install requests`

Check out [the documentation](https://docs.python-requests.org/en/master/user/install/) for more information.

**Reply by RiddyRichie** [reply]
i will do that now... thanks teju

**Comment by AdedayoAruwajoye** [comment]
Hi, I have gone through the notes and I am looking forward to understanding it better during the class tomorrow. I want to know if I'm supposed to download anything on my computer before tomorrow's class please.

**Reply by tayojabar** [reply]
Well done, Adedayo. Looking forward to helping with that tomorrow and answering other questions.

**Comment by RiddyRichie** [comment]
pls why am i getting this response from the vscode terminal:
![Screenshot (390)](https://user-images.githubusercontent.com/89553525/133020043-55e1cd8e-ac93-4b99-b17e-a53baf5d70ab.png)

**Reply by tayojabar** [reply]
Hi @RiddyRichie , this is because your terminal is currently not in the git directory on your machine. cd into `cohort7-team-Konare/cohort7-team-Konare` and then work from there. That is your git repository.

**Comment by RiddyRichie** [comment]
![Screenshot (396)](https://user-images.githubusercontent.com/89553525/135373689-ee674054-a28e-4995-9d56-01302348ad50.png)
good morning, pls I need help on how to debug the error while trying to activate. i have uninstall the vscode then install it again but still get stuck in the same error.

**Reply by tejuafonja** [reply]
Hi RiddyRichie, were you able to resolve this error? I'm not conversant with the windows OS.

**Reply by owos** [reply]
So RiddyRichie, you didn't call the file well, it is supposed to be activate.bat and remove the "." and the backward slash from the beginning.

**Reply by LaseTawak** [reply]
You're using windows, hour command is correct. Just copy the error  "you're not allowed to run scripts on your system"(something like that), paste it on Google.
You'll see how to solve it, you'll have to run a command on in powershell on powershell command prompt.
Just Google the error you'll fine the solution, it's easy to solve.

**Comment by LaseTawak** [comment]
I am just trying to set up visual studio and run the practical but I keep getting this error when I open terminal

![Screenshot (15)](https://user-images.githubusercontent.com/69163893/135708639-c19dcbb5-eb1d-4c0b-842f-b46375e1ba14.png)

**Reply by tejuafonja** [reply]
Hi Lase, did you install anaconda? https://www.anaconda.com/products/individual#Downloads

**Reply by LaseTawak** [reply]
Yes I did. I use anaconda, had it for a while.

**Reply by owos** [reply]
Hi Lase Tawak, you cannot use the Anaconda commands on any other terminal except the Anaconda prompt until the active the base evn just like the way we activated evn we created during the web scraping class.
When you activate the terminal you can then use the conda commands.
Alternatively, you could install the necessary libraries directly on your system, outside Anaconda

**Comment by LaseTawak** [comment]
Abeg, who has finished the scraper.py task ?
Need help !! 😪

**Reply by ModibboMuawiyya** [reply]
Hello am done with the Assignment, I can help you out .

**Comment by RiddyRichie** [comment]
Can you put me through also...

**Reply by ModibboMuawiyya** [reply]
Yeah sure

**Reply by ModibboMuawiyya** [reply]
Whatever you need

**Comment by LaseTawak** [comment]
@tejuafonja @theyorubayesian 
abeg this label studio isn't working for me, I've tried and tried, downloaded a lower python version and created new environment and now its giving me this error. 
I looked it up and the solutions i'm seeing are refering to anaconda and a DLL folder  
![Screenshot (17)_LI](https://user-images.githubusercontent.com/69163893/138282938-c56d8085-8aff-4167-a8ac-c268868ec240.jpg)

**Reply by owos** [reply]
Hi @LaseTawak, go and install from source. Use this link to guy you https://labelstud.io/guide/install.html#Install-from-source

first, create the evn you want the label studio to be in, 
2. activate the evn.
3. Install the label studio in the newly activated evn
I hope this helps
